src/nanoemoji/colr_to_svg.py

```diff
@@ -291,28 +307,46 @@ def _new_reuse_cache() -> ReuseCache:
     return ReuseCache(0.1, GlyphReuseCache(0.1))
 
 
-def _colr_v0_to_svgs(view_box: Rect, ttfont: ttLib.TTFont) -> Dict[str, SVG]:
+def colr_glyphs(font: ttLib.TTFont) -> Iterable[int]:
+    colr = font["COLR"]
+    if colr.version == 0:
+        for glyph_name in colr.ColorLayers:
+            yield font.getGlyphID(glyph_name)
+    else:
+        for base_glyph in font["COLR"].table.BaseGlyphList:
```

Comment:
technically in COLRv1 there could be _both_ v0 and v1 sort of color glyphs -- even though nanoemoji doesn't produce those.

Reply:
Fair point. I think explicit failure (rather than stealth broken output) might suffice for this first swing at this.

Reply:
added asserts